wiki/release/release-process.md

```diff
@@ -473,18 +473,16 @@ Steps and examples see [Pulsar Release Notes Guide](https://docs.google.com/docu
 
 ## 17. Update the site
 
-The workflow for updating the site is slightly different for major and minor releases.
-
-### Update the site for major releases
-For major release, the website is updated based on the `master` branch.
+### Update the site for minor releases
+For minor releases, such as 2.10, the website is updated based on the `master` branch.
```

Comment:
@momo-jun Why did you make this change? I think only major releases (like 2.10.0) are updated based on the `master` branch.

ping @mattisonchao @Technoboy- @codelipenghui @gaoran10

Reply:
I need to make sure I didn't understand wrong.
- major release: 2.X.0 (checkout from `master`)
- minor release: 2.X.Y (checkout from `branch-2.X`)

Reply:
@BewareMyPower This change is part of [PIP-190](https://github.com/apache/pulsar/issues/16637). 

You are right that 2.10 is updated based on the master branch. Quite often we regard it as a major version in daily talk. But it should be a minor version, while 3.0 is the next major version. 
`Pulsar version number is comprised of 3 components: major.minor.bug-fix (patch)` according to the [definition in PIP-47](https://github.com/apache/pulsar/wiki/PIP-47%3A-Time-Based-Release-Plan#what-about-version-numbers).

Reply:
As clarified by @codelipenghui and @BewareMyPower, the current consensus in the Pulsar community is:
* feature release (major release): 2.10.0, 2.11.0...
* bug-fix release (minor release): 2.10.1, 2.10.2...
I will update the wording `minor` accordingly.